### PR TITLE
Update deploy-on-gcp-gke.md (#1022)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #1022 to release-1.1
You can switch your code base to this Pull Request by using [git-extras](https://github.com/tj/git-extras):
```bash
# In docs-tidb-operator repo:
git pr https://github.com/pingcap/docs-tidb-operator/pull/1039
```

After apply modifications, you can push your change to this PR via:
```bash
git push git@github.com:ti-srebot/docs-tidb-operator.git pr/1039:release-1.1-2198bb781cf3
```

---

Added reference link for tidb-cluster CR and API

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/deploy-on-gcp-gke.md [hunk 1]
```diff
@@ -91,13 +91,29 @@ kubectl create namespace tidb-cluster
...
+```
+
+Refer to [configure the TiDB cluster](configure-a-tidb-cluster.md) to further customize and configure the CR before applying.
+
 To deploy the `TidbCluster` and `TidbMonitor` CR in the GKE cluster, run the following command:
 
 {{< copyable "shell-regular" >}}
 
 ```shell
+<<<<<<< HEAD
```
Comment (ran-huang) [inline en/deploy-on-gcp-gke.md:110]: @DanielZhangQD PTAL

File: en/deploy-on-gcp-gke.md [hunk 2]
```diff
@@ -91,13 +91,29 @@ kubectl create namespace tidb-cluster
...
 kubectl create -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.1.6/examples/gcp/tidb-cluster.yaml -n tidb-cluster &&
 kubectl create -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.1.6/examples/gcp/tidb-monitor.yaml -n tidb-cluster
+=======
+kubectl create -f tidb-cluster.yaml -n tidb-cluster && \
+kubectl create -f tidb-monitor.yaml -n tidb-cluster
+>>>>>>> 2198bb7... Update deploy-on-gcp-gke.md (#1022)
```
Comment (ran-huang) [inline en/deploy-on-gcp-gke.md:116]:
```suggestion
kubectl create -f tidb-cluster.yaml -n tidb-cluster && \
kubectl create -f tidb-monitor.yaml -n tidb-cluster
```

**Review by ran-huang — APPROVED** [review]

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Pull request merged**